### PR TITLE
Db layout

### PR DESCRIPTION
**Pull request by baudot**

### CODE REVIEW

File: db/reset_db.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+dropdb stonks
```
Comment (NylonEric) [inline db/reset_db.sh:2]: lol

File: db/setup.sql [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  ticker_symbol varchar(8) NOT NULL,
+  exchange varchar(8) NOT NULL
+);
+
+CREATE INDEX ON watchlist (user_id);
+CREATE INDEX ON watchlist (ticker_symbol, exchange);
+
+-- Sample values.
+
+INSERT INTO users
```
Comment (NylonEric) [inline db/setup.sql:63]: I would say separate out these mock insert queries into a separate file for production, but it's fine for our purposes in MVP.

**Review by minongcopper — APPROVED** [review]
I am assuming that there is a good reason that Sam picked bigint over money as the data type for cash_position (given that Sam is the DB wizard among us). I would , in the real world, like to see more in the user table regarding things like total deposits/withdrawals, so as to give better opportunity to develop a more sophisticated ranking formula than just cash_position. But other than those pieces, I think this db structure is on great footing! Thanks for the hard work.

**Review by NylonEric — APPROVED** [review]

**Review by tbrinkman3 — APPROVED** [review]
Thanks for the reset script!

**Comment by baudot** [comment]
> I am assuming that there is a good reason that Sam picked bigint over money as the data type for cash_position (given that Sam is the DB wizard among us). I would , in the real world, like to see more in the user table regarding things like total deposits/withdrawals, so as to give better opportunity to develop a more sophisticated ranking formula than just cash_position. But other than those pieces, I think this db structure is on great footing! Thanks for the hard work.

 I probably SHOULD have used a monetary type rather than bigint. Not all of my db reflexes are good! Thanks for calling me on it,

**Pull request merged**